src/Microsoft.Extensions.Logging.Abstractions/Internal/LogValuesFormatter.cs

```diff
@@ -114,6 +115,8 @@ private static int FindIndexOf(string format, char ch, int startIndex, int endIn
             return findIndex == -1 ? endIndex : findIndex;
         }
 
+        private static readonly object[] EmptyArray = new object[0];
```

Comment:
btw, is the `s_` variable name prefix for static/const privates `s_varName` a convention we do on this team?